openmc/mgxs/mgxs.py

```diff
@@ -1570,10 +1572,27 @@ def tallies(self):
     @property
     def rxn_rate_tally(self):
         if self._rxn_rate_tally is None:
-            self._rxn_rate_tally = self.tallies['nu-fission']
+            self._rxn_rate_tally = self.tallies[self._rxn_type]
             self._rxn_rate_tally.sparse = self.sparse
         return self._rxn_rate_tally
 
+class NuFissionXS(FissionXSBase):
+    """A fission product multi-group cross section."""
+
+    def __init__(self, domain=None, domain_type=None,
+                 groups=None, by_nuclide=False, name=''):
+        super(NuFissionXS, self).__init__('nu-fission', domain, domain_type,
+                                          groups, by_nuclide, name)
+        pass
```

Comment:
Nope - I'll remove them.  I've been doing too much C++ lately... so my brain was considering the `super` to be part of the "initialization list" and it thought the body of the function was empty so my fingers put a `pass` in there ;-)